node_modules/col-canvas/lib/poller.js

```diff
@@ -204,8 +214,11 @@ var handleCourse = module.exports.handleCourse = function(course, opts, callback
           }
 
           // If the course has activity and that activity is in the distant past, we continue to the deactivation step; otherwise return.
```

Comment:
@paulkerschen nit-picky question: The comment above ("...otherwise return") describes the return at line 222. Maybe move the comment down to avoid confusion with line 218?